torch/overrides.py

```diff
@@ -167,6 +167,7 @@ def get_ignored_functions() -> Set[Callable]:
         torch.nn.functional.upsample_bilinear,
         torch.nn.functional.upsample_nearest,
         torch.nn.functional.has_torch_function,
+        torch.nn.functional.object_has_torch_function,
         torch.nn.functional.handle_torch_function,
```

Comment:
??? Are these getting secretly exported when they shouldn't be ???

Reply:
I was surprised too, but yeah. Tests fail if I don't.

Reply:
Can you file an issue about this? They shouldn't be exported... you might be able to fix this problem by importing the module rather than the functions; maybe `from blah import foo` implicitly sets up an export.

Reply:
https://github.com/pytorch/pytorch/issues/49480